core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowLabels.scala

```diff
@@ -16,6 +16,8 @@
 
 package cloudflow.operator
 
+import cloudflow.blueprint.deployment.CloudflowApplication
```

Comment:
This can probably cleaned up (I know you are still cleaning up, so just leaving some very minor remarks)

Reply:
done ..